pkg/daemon/ceph/client/config.go

```diff
@@ -302,3 +302,31 @@ func WriteCephConfig(context *clusterd.Context, clusterInfo *ClusterInfo) error
 	}
 	return nil
 }
+
+// SetConfig applies a setting for a single mgr daemon
+func SetConfig(context *clusterd.Context, clusterInfo *ClusterInfo, daemonID string, key, val string, force bool) (bool, error) {
+	var getArgs, setArgs []string
+	getArgs = append(getArgs, "config", "get", daemonID, key)
+	if val == "" {
+		setArgs = append(setArgs, "config", "rm", daemonID, key)
+	} else {
+		setArgs = append(setArgs, "config", "set", daemonID, key, val)
+	}
+	if force {
+		setArgs = append(setArgs, "--force")
+	}
+
+	// Retrieve previous value to monitor changes
+	var prevVal string
+	buf, err := NewCephCommand(context, clusterInfo, getArgs).Run()
+	if err == nil {
+		prevVal = strings.TrimSpace(string(buf))
+	}
+
+	if _, err := NewCephCommand(context, clusterInfo, setArgs).Run(); err != nil {
+		return false, errors.Wrapf(err, "failed to set config key %s to \"%s\"", key, val)
```

Comment:
```suggestion
		return false, errors.Wrapf(err, "failed to set config key %q to %q", key, val)
```

I thought you said `%q` was taking over the world? 😄

Reply:
I'm innocent, I just moved that code and didn't just add it. :)